tests/__fixtures__/performance.json

```diff
@@ -1,7 +1,7 @@
 {
     "id": "83d526a5d20040a7b854a2cbef64c067",
     "name": "my performance test",
-    "url": "https://app.saucelabs.com/tests/83d526a5d20040a7b854a2cbef64c067",
+    "url": "https://app.saucelabs.com/performance/83d526a5d20040a7b854a2cbef64c067",
```

Comment:
So I don't think this works.  If you go to https://app.saucelabs.com/performance/83d526a5d20040a7b854a2cbef64c067 you just get a blank page because I think it expects the index number in the url as well.  e.g. https://app.saucelabs.com/performance/83d526a5d20040a7b854a2cbef64c067/0

I think defaulting to the first url is fine (/0)

Reply:
Good catch